apps/ern-skin/src/views/view-documents.vue

```diff
@@ -15,8 +15,80 @@
       :verticalPadding="2"
     >
       <h2 id="section-documents-title"></h2>
-      <p>Download additional information about the ERRAS Registry.</p>
-      <FileList table="Files" labelsColumn="name" fileColumn="file" />
+      <div>
+        <button
+          style="
+            background-color: #0084b4;
+            border-color: #0084b4;
+            color: #ffffff;
+          "
+          @click="toggleStudies"
+        >
+          {{ studiesOpen ? "Studies" : "Studies" }}
+        </button>
+        <ul v-if="studiesOpen">
+          <Studies table="Studies" labelsColumn="title" />
+        </ul>
+      </div>
+      <div>
+        <button
+          style="
+            background-color: #00a453;
+            border-color: #00a453;
+            color: #ffffff;
+          "
+          @click="toggleAccess"
+        >
+          {{ accessOpen ? "Access Requests" : "Access Requests" }}
```

Comment:
Does the label need to be changed when the content is expanded? If not the label can be-

```vue
<button ...>
    Access Requests
</button>
```